numpy/lib/function_base.py

```diff
@@ -1867,13 +1867,24 @@ def diff(a, n=1, axis=-1):
         The number of times values are differenced.
     axis : int, optional
         The axis along which the difference is taken, default is the last axis.
+    to_begin : 1d array_like
+        Values to insert along axis before the differenced results.  'first' is
+        an alias for a.take([0], axis), which inserts the first value from a 
+        along the specified axis, which is the inverse of cumsum.  Scalar values 
+        are broadcast across all other axes.  Otherwise the dimension and shape 
```

Comment:
"Scalar values are broadcast across all other axes" isn't *quite* the full story. Maybe "Scalar values are  expanded to arrays with length 1 in the direction of axis and broadcast across all other axes"?